js/app.js

```diff
@@ -0,0 +1,76 @@
+/* Theme toggle */
+const themeStyle = document.querySelector('.themestyle');
+let activeSheet = document.querySelector("#active-stylesheet");
+let themeToggle = document.querySelector(".theme-toggle");
+let themeIcon = document.querySelector('.theme-toggle i');
+
+if (localStorage.getItem("lastActiveSheet")) {
+    activeSheet.setAttribute("href", localStorage.getItem("lastActiveSheet"));
+}
+
+function switchStyle() {
+    let selectedSheet = this.getAttribute("data-stylesheet");
+    console.log(selectedSheet);
```

Comment:
Suggestion: function is large, could add javascript docstring. 

Demo:
---
![image](https://user-images.githubusercontent.com/91944618/154958640-640869b8-1ec3-4264-acba-eed0a6a73f13.png)

Here's some references to read more on that if you need,
https://jsdoc.app/about-getting-started.html
https://dev.to/stephencweiss/write-your-own-javascript-contracts-and-docstrings-42ho